src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -542,11 +547,12 @@ public function longText($column)
      * @param  string  $column
      * @param  bool  $autoIncrement
      * @param  bool  $unsigned
+     * @param  int  $auto_increment
      * @return \Illuminate\Support\Fluent
      */
-    public function integer($column, $autoIncrement = false, $unsigned = false)
+    public function integer($column, $autoIncrement = false, $unsigned = false, $auto_increment = 1)
```

Comment:
@GrahamCampbell but then we conflict with the existing one. Any suggestions? 😔

Reply:
I don't think it should be here anyway, but ` $autoIncrementValue`?